daemon/daemon.go

```diff
@@ -880,9 +877,18 @@ func NewDaemonFromDirectory(config *Config, eng *engine.Engine) (*Daemon, error)
 	}
 	log.Debugf("Using graph driver %s", driver)
 
-	// As Docker on btrfs and SELinux are incompatible at present, error on both being enabled
-	if selinuxEnabled() && config.EnableSelinuxSupport && driver.String() == "btrfs" {
-		return nil, fmt.Errorf("SELinux is not supported with the BTRFS graph driver!")
+	if config.EnableSelinuxSupport {
+		if selinuxEnabled() {
+			// As Docker on btrfs and SELinux are incompatible at present, error on both being enabled
+			if driver.String() == "btrfs" {
+				return nil, fmt.Errorf("SELinux is not supported with the BTRFS graph driver")
+			}
+			log.Debugf("SELinux enabled successfully")
+		} else {
+			log.Warnf("SELinux could not enable on the host system. Docker is not able to enable SELinux")
```

Comment:
current log info is followed to rhatdan's suggestion.
"SELinux could not enable on the host system" means that Selinux did not enable on the host system, "Docker is not able to enable SELinux" means that Docker enabled Selinux failure.In my opinion,maybe we could keep current situation,what do you think?